diff.py

```diff
@@ -524,9 +532,7 @@ def create_config(args: argparse.Namespace, project: ProjectSettings) -> Config:
         max_function_size_bytes=args.max_lines * 4,
         # Display options
         formatter=formatter,
-        threeway=args.threeway,
-        zeroway=args.zeroway,
-        oneway=args.oneway,
+        diff_type=DiffType(args.diff_type or DiffType.NORMAL),
```

Comment:
can the argparse `const` values be enum members already, so you don't have to do this fallible str -> enum conversion?